erpnext/accounts/doctype/tax_withholding_entry/tax_withholding_entry.py

```
@@ -0,0 +1,1414 @@
+				"taxable_amount": category.taxable_amount,
+				"taxable_doctype": "",
+				"taxable_name": "",
+				"taxable_date": "",
+			}
+		)
+
+		open_entries["over_withheld"].append(current_entry)
+
+		return open_entries
+
+	def _is_threshold_crossed_for_category(self, category):
```

Ruff (0.14.8)

Unused method argument: `category`

(ARG002)